pkg/proc/arm64_arch.go

```diff
@@ -258,6 +376,7 @@ func (a *ARM64) RegistersToDwarfRegisters(staticBase uint64, regs Registers) op.
 	dregs[arm64DwarfIPRegNum] = op.DwarfRegisterFromUint64(regs.PC())
 	dregs[arm64DwarfSPRegNum] = op.DwarfRegisterFromUint64(regs.SP())
 	dregs[arm64DwarfBPRegNum] = op.DwarfRegisterFromUint64(regs.BP())
+	dregs[arm64DwarfLRRegNum] = op.DwarfRegisterFromUint64(regs.LR())
```

Comment:
Instead of adding `LR()` to the overall Registers interface, can we just type cast once within this function and access LR directly? I'd like to avoid adding all the methods that simply return 0 if possible, and unless I missed something that method is only used here where we would always expect the ARM register struct.

Reply:
Removed.